### DISCUSSION TITLE
Resizing on touchpad

### DISCUSSION THREAD

**Discussion by patrickaldis** [hyprwm/Hyprland#618] [General]
I think the resizing experience is great with an external mouse, but when just using a touchpad, right-click and drag becomes really awkward.

Just wondering what other options there could be

It could make sense to be able to KEY+drag to resize?

**Comment by vaxerski** [comment]
it's planned

**Reply by patrickaldis** [reply]
Oh awesome. Much love

**Reply by patrickaldis** [reply]
think this can be closed now

**Reply by vaxerski** [reply]
this is a discussion, not an issue. You don't just close discussions

**Comment by buttergrillcorn** [comment]
Try add this in your config file:

`bindm=SUPER_CTRL, mouse:272, resizewindow`

You basically just have to press Super+Ctrl then click(tap) and drag on your touch pad to resize.

It would be better if there's a way to use keybindings and just simply swipe on touch pad to resize. If you or anyone knows a way to do this please let me know, thanks. But for now this sort of works pretty okay.

Hope that helps :)

**Reply by JulioNectar** [reply]
**If you want to use only touchpad to resize (without using the right click button), you can tap touchpad with two fingers and quickly press one finger, then move it.**

**Reply by aviral-public** [reply]
@JulioNectar thanks man if I didn't come across this comment I'd have gone mad

**Reply by marcellogmoreira** [reply]
> **If you want to use only touchpad to resize (without using the right click button), you can tap touchpad with two fingers and quickly press one finger, then move it.**

this is pure bliss, i was about to go mad, thanks

**Reply by j-ameswong** [reply]
> **If you want to use only touchpad to resize (without using the right click button), you can tap touchpad with two fingers and quickly press one finger, then move it.**

This is a godsend, how did you even figure this out?

**Reply by JulioNectar** [reply]
> > **If you want to use only touchpad to resize (without using the right click button), you can tap touchpad with two fingers and quickly press one finger, then move it.**
> 
> This is a godsend, how did you even figure this out?

Glad this still useful. I figure it out accidentally. But I think it's a bug, because even the dev said that this feature was planned.
